src/chainstate/burn/db/sortdb.rs

```diff
@@ -992,6 +1017,92 @@ impl <'a> SortitionHandleConn <'a> {
         Ok(key_status)
     }
 
+    /// Return a vec of sortition winner's burn header hash and stacks header hash, ordered by
+    ///   increasing block height in the range (block_height_begin, block_height_end]
+    fn get_sortition_winners_in_fork(&self, block_height_begin: u32, block_height_end: u32) -> Result<Vec<(Txid, u64)>,  BurnchainError> {
+        let mut result = vec![];
+        for height in (block_height_begin+1)..(block_height_end+1) {
+            debug!("Looking for winners at height = {}", height);
+            let snapshot = SortitionDB::get_ancestor_snapshot(self, height as u64, &self.context.chain_tip)?
+                .ok_or_else(|| BurnchainError::MissingParentBlock)?;
+            if snapshot.sortition {
+                result.push((snapshot.winning_block_txid, snapshot.block_height));
+            }
+        }
+        Ok(result)
+    }
+
+    pub fn get_reward_cycle_info(&self, prepare_end: &BurnchainHeaderHash, pox_consts: &PoxConstants) -> Result<Option<(ConsensusHash, BlockHeaderHash)>, CoordinatorError> {
```

Comment:
Isn't this method just returning the ID of the anchor block selected as of `prepare_end`, instead of a `RewardCycleInfo` struct?  If so, maybe this method should be called `get_anchor_block_id` so it won't confuse readers who see `get_reward_cycle_info` used in the chain cooordinator?

Reply:
Sure, I'll update it to `get_pox_anchor_block`, and add some rustdocs.